docs/quickstart.md

```diff
@@ -179,16 +185,14 @@ TCP and a Unix socket
 
 ## Committing (saving) a container state
 
-Save your containers state to an image, so the state can be
-re-used.
-
-When you commit your container, Docker only stores the diff (difference) between
-the source image and the current state of the container's image. To list images
-you already have, use the `docker images` command.
-
+To save the current state of a container as an image, run:
```

Comment:
Can you add a blank line after `run:`? Haven't tested, but I think it's needed before a code block